viewcontroller/converter/converter.go

```diff
@@ -0,0 +1,58 @@
+package converter
```

Comment:
if you copied any of the methods here, please add a comment with credit

Reply:
no